src/Core/Entity/Permission.php

```diff
@@ -15,37 +15,38 @@
 
 class Permission extends StaticEntity
 {
-	protected $id;
-	protected $name;
-	protected $description;
-
-	// Standard permissions names
-    const DATA_IMPORT          = 'Bulk Data Import and Export';
-    const MANAGE_POSTS         = 'Manage Posts';
-    const MANAGE_SETS          = 'Manage Collections and Saved Searches';
-    const MANAGE_SETTINGS      = 'Manage Settings';
-    const MANAGE_USERS         = 'Manage Users';
-    const EDIT_OWN_POSTS       = 'Edit their own posts';
-
-	// DataTransformer
-	public function getDefinition()
-	{
-		return [
-			'id'           => 'int',
-			'name'         => 'string',
-			'description'  => 'string',
-		];
-	}
-
-	// Entity
-	public function getResource()
-	{
-		return 'permission';
-	}
-
-	// StatefulData
-	protected function getImmutable()
-	{
-		return array_merge(parent::getImmutable(), ['name']);
-	}
+    protected $id;
+    protected $name;
+    protected $description;
+    // FIXME: this LEGACY_DATA_IMPORT has to be removed after the prod release
+    const LEGACY_DATA_IMPORT    = 'Bulk Data Import';
```

Comment:
I will remove LEGACY_DATA_IMPORT after the release is tested and complete, and do another release with the correct DATA_IMPORT_EXPORT permission only